src/main.zig

```diff
@@ -233,43 +284,47 @@ fn initLogging() void {
 		std.log.err("Couldn't create {s}: {s}", .{_path_str, @errorName(err)});
 		return;
 	};
+	logFileTsWriter = logFileTs.?.writer(&logFileTsBuffer);
 
 	supportsANSIColors = std.fs.File.stdout().supportsAnsiEscapeCodes();
 }
 
 fn deinitLogging() void {
 	if(logFile) |_logFile| {
+		logFileWriter.interface.flush() catch {};
+		logFileBuffer = undefined;
```

Comment:
I understand for the writer, but there is nothing gained by setting the byte buffer to undefined.